metro.config.js

```diff
@@ -1,28 +1,16 @@
-// exclusionList is a function that takes an array of regexes and combines
-// them with the default exclusions to return a single regex.
-// please see: https://stackoverflow.com/a/41963217/8791773 for more info
-const exclusionList = require('metro-config/src/defaults/exclusionList');
-const {components} = require('./export.config.json');
-const {buildDir} = require('./buildConstants');
+const {getDefaultConfig} = require('metro-config');
```

Comment:
@SergeySychevskiy This might break smt, idk what :)

Reply:
Neither do I :D

Reply:
Do you really need to change metro config?
Since you just take `svg` files from node_modules - do you still need these changes?
Because I don't really see the reason for them, and I guess it might break the build

Reply:
tested yarn run build and worked